core/src/main.cpp

```diff
@@ -267,6 +268,12 @@ constexpr auto default_jsonschema() -> std::string_view
                                         "type": "object"
                                     }},
                                     "minProperties": 1
+                                }},
+                                "mapping_lib": {{
```

Comment:
definitely discush...

```suggestion
                                "user_mapping_shared_library": {{
```

Reply:
Updated, leaving open if name change is desired.